ksqldb-functional-tests/src/test/resources/query-validation-tests/fk-join.json

```diff
@@ -268,6 +268,482 @@
         "type": "io.confluent.ksql.util.KsqlStatementException",
         "message": "Invalid join condition: foreign-key table-table joins are not supported as part of n-way joins. Got LEFT_TABLE.F1 = RIGHT_TABLE.ID3."
       }
+    },
+    {
+      "name": "value column - inner join",
+      "properties": {
+        "ksql.joins.foreign.key.enable": true
+      },
+      "statements": [
+        "CREATE TABLE left_table (L_ID BIGINT PRIMARY KEY, NAME varchar, VALUE bigint) WITH (kafka_topic='left_topic', value_format='JSON');",
```

Comment:
As mentioned elsewhere: the formatting make it really hard to read. We should use all-upper-case only for _keywords_, an should use all-lower for identifiers... (especially the wild mix of all-lower / all-upper is super confusing)

Reply:
I'll keep this in mind for new tests but I don't think it's worth going back and updating this for existing tests. If we really want to, we can change this after we ship the feature.